src/essence_parser/RawEssenceReader.cpp

```diff
@@ -53,9 +53,10 @@ using namespace std;
 using namespace bmx;
 
 
-#define READ_BLOCK_SIZE         8192
-#define PARSE_FRAME_START_SIZE  8192
-
+//#define READ_BLOCK_SIZE         8192
+//#define PARSE_FRAME_START_SIZE  8192
+#define READ_BLOCK_SIZE         3000000
+#define PARSE_FRAME_START_SIZE  3000000   // 156556800
```

Comment:
Yes, this was an optimization for JPEG XS since the frame size is in general much larger. I have included another setter for this and left 8192 as default.